leakpro/attacks/mia_attacks/abstract_mia.py

```diff
@@ -53,6 +57,7 @@ def __init__(
                 # out_members will start after the last training index and go up to the number of test indices - 1
                 "out_members": np.arange(len(handler.train_indices),len(handler.train_indices)+len(handler.test_indices)),
             }
+            AbstractMIA.skip_indices = np.zeros(len(AbstractMIA.audit_dataset["data"]), dtype=bool)
```

Comment:
Should this really be shared among all MIAs? Right now we have used it for RMIA and LiRA but they may not use the same number of shadow models and then these indices will not be the same.

Reply:
Since memorization is moved from abstract_mia.py and we have another way to handle 0 IN-sample and 0 OUT-sample cases, this has been removed